job/models.py

```diff
@@ -2758,6 +2790,532 @@ class Message(BaseModel, models.Model):
+    )
+
+    can_supply_steam_turbine = models.BooleanField(
+        default=True,
+        blank=True,
+        null=True,
+        help_text="If the boiler can supply steam to the steam turbine for electric production"
+    )
+
+    # For custom validations within model.
+    def clean(self):
+        self.fuel_cost_per_mmbtu = scalar_to_vector(self.fuel_cost_per_mmbtu)
+
+class BoilerOutputs(BaseModel, models.Model):
+
+    key = "Boiler"
+
+    meta = models.OneToOneField(
+        APIMeta,
+        on_delete=models.CASCADE,
+        related_name="BoilerOutputs",
+        primary_key=True
+    )
+
+    year_one_fuel_consumption_mmbtu = models.FloatField(null=True, blank=True)
+
+    year_one_fuel_consumption_mmbtu_per_hour = ArrayField(
+        models.FloatField(null=True, blank=True),
+        default=list,
+    )
+
+    lifecycle_fuel_cost = models.FloatField(null=True, blank=True)
+    lifecycle_per_unit_prod_om_costs = models.FloatField(null=True, blank=True)
+    lifecycle_fuel_cost_bau = models.FloatField(null=True, blank=True)
+    year_one_thermal_production_mmbtu = models.FloatField(null=True, blank=True)
+    year_one_fuel_cost = models.FloatField(null=True, blank=True)
+    
+    thermal_to_tes_series_mmbtu_per_hour = ArrayField(
+        models.FloatField(null=True, blank=True),
+        default = list,
+    )
+    year_one_thermal_production_mmbtu_per_hour = ArrayField(
+        models.FloatField(null=True, blank=True),
+        default = list,
+    )
+    year_one_thermal_to_load_series_mmbtu_per_hour = ArrayField(
+        models.FloatField(null=True, blank=True),
+        default = list,
+    )
+
+
+class SpaceHeatingLoadInputs(BaseModel, models.Model):
+    
+    key = "SpaceHeatingLoad"
+
+    meta = models.OneToOneField(
+        APIMeta,
+        on_delete=models.CASCADE,
+        related_name="SpaceHeatingLoadInputs",
+        primary_key=True
+    )
+
+    possible_sets = [
+        ["fuel_loads_mmbtu_per_hour"],
+        ["doe_reference_name", "monthly_mmbtu"],
+        ["annual_mmbtu", "doe_reference_name"],
+        ["doe_reference_name"],
+        ["blended_doe_reference_names", "blended_doe_reference_percents"]
+    ]
+
+    DOE_REFERENCE_NAME = models.TextChoices('DOE_REFERENCE_NAME', (
+        'FastFoodRest '
+        'FullServiceRest '
+        'Hospital '
+        'LargeHotel '
+        'LargeOffice '
+        'MediumOffice '
+        'MidriseApartment '
+        'Outpatient '
+        'PrimarySchool '
+        'RetailStore '
+        'SecondarySchool '
+        'SmallHotel '
+        'SmallOffice '
+        'StripMall '
+        'Supermarket '
+        'Warehouse '
+        'FlatLoad '
+        'FlatLoad_24_5 '
+        'FlatLoad_16_7 '
+        'FlatLoad_16_5 '
+        'FlatLoad_8_7 '
+        'FlatLoad_8_5'
+    ))
+
+    annual_mmbtu = models.FloatField(
+        validators=[
+            MinValueValidator(1),
+            MaxValueValidator(100000000)
```

Comment:
update `MaxValueValidator `value to `MAX_BIG_NUMBER` here as well

Reply:
addressed in https://github.com/NREL/REopt_API/pull/322/commits/8b9bde382331eb675af40fb1f85d1abf97d53d49